blocks/api/paste.js

```diff
@@ -76,7 +78,9 @@ export function normaliseToBlockLevelNodes( nodes ) {
 }
 
 export default function( nodes ) {
-	return normaliseToBlockLevelNodes( nodes ).map( ( node ) => {
+	const prepare = compose( [ normaliseToBlockLevelNodes, gDocs, stripAttributes ] );
```

Comment:
Okay, so for now we just run all of them? I think it would be good to get some mechanism in place already, because I think we'll need it eventually. E.g. I don't think we want to run MarkDown processors on all content? Or _do_ we?